### ISSUE TITLE
Root#remove is deprecated. Use Root#removeChild

### ISSUE THREAD

**Issue by oleersoy** [postcss/postcss#611]
Hello,

I'm using the [postcss-css-variables](https://github.com/MadLittleMods/postcss-css-variables) module.  I have a very simple setup.

input.css

``` css
:root {
  --varA: 'value';
}
```
process.js

``` js
var postcss = require('postcss');
var cssvariables = require('postcss-css-variables');

var fs = require('fs');

var mycss = fs.readFileSync('input.css', 'utf8');

// Process your CSS with postcss-css-variables
var output = postcss([
        cssvariables(/*options*/)
    ])
    .process(mycss)
    .css;

console.log(output);
```

When I run:
`node process.js` I get:

``` console
ole@MKI:~/Sandbox/test1$ node process.js 
Root#remove is deprecated. Use Root#removeChild
/home/ole/Sandbox/test1/node_modules/postcss/lib/lazy-result.js:204
            throw error;
            ^

TypeError: Cannot set property 'parent' of undefined
    at Root.removeChild (/home/ole/Sandbox/test1/node_modules/postcss/lib/container.js:344:34)
    at Root.removeChild (/home/ole/Sandbox/test1/node_modules/postcss/lib/root.js:37:49)
    at Root.remove (/home/ole/Sandbox/test1/node_modules/postcss/lib/root.js:87:14)
    at cleanUpNode (/home/ole/Sandbox/test1/node_modules/postcss-css-variables/index.js:45:16)
    at Array.forEach (native)
    at /home/ole/Sandbox/test1/node_modules/postcss-css-variables/index.js:242:22
    at LazyResult.run (/home/ole/Sandbox/test1/node_modules/postcss/lib/lazy-result.js:201:20)
    at LazyResult.sync (/home/ole/Sandbox/test1/node_modules/postcss/lib/lazy-result.js:188:32)
    at LazyResult.stringify (/home/ole/Sandbox/test1/node_modules/postcss/lib/lazy-result.js:212:14)
    at LazyResult.get (/home/ole/Sandbox/test1/node_modules/postcss/lib/lazy-result.js:241:25)


```

Thoughs?

Ole

**Comment by oleersoy** [comment]
If I modify the input.css file to use the variable, as follows:

```css
:root {
  --varA: 'value';
}

h1 {

  color: var(--varA);
}
```

Then there are no exceptions and the correct result gets logged:

``` css
ole@MKI:~/Sandbox/test1$ node process.js 
h1 {

  color: 'value';
}


```

**Comment by TrySound** [comment]
@oleersoy Which versions of modules do you use?

**Comment by oleersoy** [comment]
"postcss": "^5.0.10",
    "postcss-css-variables": "^0.5.0"

**Comment by ai** [comment]
It is issue of postcss-css-variables, it uses old API. So I close this issue.

**Comment by MadLittleMods** [comment]
Hey @oleersoy, thank you for the report.

I have released a fix for this in [`v0.5.1`](https://github.com/MadLittleMods/postcss-css-variables/blob/master/CHANGELOG.md#v051---2015-10-24) and added a [test](https://github.com/MadLittleMods/postcss-css-variables/blob/c6197e1a0da4ed2e21e78b0002eb610cd00ada7d/test/test.js#L64) to make sure we don't regress.

**Comment by oleersoy** [comment]
Great thanks @MadLittleMods!
